src/routes/userData.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+const userService = require('../services/userData');
+const roverPhotoService = require('../services/roverPhotos');
+const UserRequestDto = require('../dto/userRequestDto');
+
+router.post('/', async function (req, res, next) {
+  const userData = new UserRequestDto(req.body);
```

Comment:
point for improvements: ideally It is better to validate data in the middleware before the controller. If the data is not valid, throw an exception

Reply:
Will implement validation as a part of future work (I saw there is a task for this)